documentation/troubleshooting.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Troubleshooting Kubernetes vSphere Cloud Provider
+---
+To troubleshoot issues with Kuberenetes cluster deployment, please visit [https://kubernetes.io/docs/tasks/debug-application-cluster/debug-cluster/](https://kubernetes.io/docs/tasks/debug-application-cluster/debug-cluster/)
+
+## Logs to inspect
+To debug vSphere Cloud Provider Issues, Controller-Manager, API-server and Kubelet logs should be checked.
+
+Logs for Controller-Manager and API server should be available on the Master node VM and Logs for Kubelet is located on both Master and Worker Nodes.
```

Comment:
Let us move this to relevant sections i.e. API server logs and Controller manager logs.